Sources/NIO/Interfaces.swift

```diff
@@ -0,0 +1,97 @@
+//
```

Comment:
@Lukasa this needs the license header....

Reply:
ACK.